src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/UpdateUsersFromApiSeedJob.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.seed
+
+import org.slf4j.LoggerFactory
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.model.ServiceName
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.service.UserService
+import java.util.UUID
+
+class UpdateUsersFromApiSeedJob(
+  fileName: String,
+  val userService: UserService,
+) : SeedJob<UpdateUserFromApiCsvRow>(
+  id = UUID.randomUUID(),
+  fileName = fileName,
+  requiredHeaders = setOf(
+    "delius_username",
+    "service_name",
+  ),
+) {
+  private val log = LoggerFactory.getLogger(this::class.java)
+
+  override fun deserializeRow(columns: Map<String, String>) = UpdateUserFromApiCsvRow(
+    deliusUsername = columns["delius_username"]!!.trim(),
```

Comment:
Is there an example CSV file/are there instructions for running this job?

Reply:
We have some generic instructions here that should be sufficient - https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/doc/how-to/run_seed_job_remotely.md

I think it's probably a good idea to document all the seed jobs we have somewhere (confluence?) and at the same time remove some of the older jobs that were for 'one off' jobs, or no longer have a use.